wom/src/main/scala/wom/package.scala

```diff
@@ -31,6 +31,13 @@ package object values {
     def md5Sum: String = DigestUtils.md5Hex(value)
     def md5SumShort: String = value.md5Sum.substring(0, 8)
   }
+
+  implicit class ShellQuoteHelper(val s: String) extends AnyVal {
+    // Escape any double quote characters in the string and surround with double quotes. The CWL spec doesn't
```

Comment:
I'm curious about this relocation. This seems more `CWL` than `WOM` to me.

Reply:
I actually didn't need to move this as part of these changes but it seemed like a good idea. As the comments say this escapes double quotes and puts double quotes around strings, which is useful for elements of a command instantiation. We should probably be using something like this more liberally to support filenames with spaces.